source/IntegrationEventListener/Energinet.DataHub.Aggregations.DatabaseMigration/Scripts/202203250902_Create_Charge_Table.sql

```diff
@@ -0,0 +1,18 @@
+﻿CREATE TABLE [dbo].[Charge]
+(
+	[RowId] [uniqueidentifier] NOT NULL,
+	[ChargeKey] [nvarchar](max) NOT NULL,
+	[ChargeId] [nvarchar](255) NOT NULL,
+	[ChargeType] [int] NOT NULL,
+	[ChargeOwner] [nvarchar](255) NOT NULL,
+	[Resolution] [int] NOT NULL,
+	[ChargeTax] [bit] NOT NULL,
```

Comment:
This will align naming with charge domain:
```suggestion
	[TaxIndicator] [bit] NOT NULL,
```